Dockerfile

```diff
@@ -15,4 +15,4 @@ WORKDIR /app
 
 COPY --from=composer /app /app
 
-CMD ["php", "-S", "0.0.0.0:5050", "./__tests__/index.php"]
+CMD ["php", "vendor/bin/codecept", "run", "Unit"]
```

Comment:
W readme jest informacja, żeby odpalac to z poleceniem
`3. Run `docker run -p 5050:5050 --rm php` to run the tests and delete container immediately after.`
Więc to raczej do zmiany bo nie trzeba robić mapowania portów.